jupyter_firefly_extensions/__init__.py

```diff
@@ -51,6 +81,7 @@ def load_jupyter_server_extension(nb_server_app):
     Args:
         nb_server_app (NotebookWebApplication): handle to the Notebook webserver instance.
     """
+    logger = logging.getLogger(__name__)
```

Comment:
Recommend to move to top level per the comment on the `get()` method

Reply:
ok